views/delete.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= @product.name %></h1>
+Some product info<br>
+<form action="/<%=@product.id%>/delete" method="post">
```

Comment:
I would follow the convention with keeping spacebars between these brackets and text inside them, as you used above in line 1 `<% ... %>`. You might want to correct views/index.erb and views/product.erb in the same way ;)